docs/examples/paddle/paddle-basic_example.ipynb

```diff
@@ -0,0 +1,151 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using DALI in PaddlePaddle\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This example shows how to use DALI in PaddlePaddle.\n",
+    "\n",
+    "This example uses CaffeReader.\n",
+    "See other [examples](...) for details on how to use different data formats."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us start from defining some global constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Caffe LMDB\n",
+    "lmdb_folder = \"/data/imagenet/train-lmdb-256x256\"\n",
```

Comment:
It would be better to use DALI_extra, like in https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/pytorch/pytorch-various-readers.html.

Reply:
done.